docs/book/content/calibration/demographics.md

```diff
@@ -183,51 +157,37 @@ We discuss the approach to estimating fertility rates $f_s$, mortality rates $\r
       infmort_rates=None,
       pop_dist=None,
       country_id="710",
-      start_year=2023,
-      end_year=2023,
-      graph=False,
+      start_year=YEAR_TO_PLOT,
+      end_year=YEAR_TO_PLOT + 50,
+      graph=True,
       plot_path=None,
       download_path=None,
   )
-
-  plt.plot(
-      np.arange(1, 101), np.squeeze(imm_rates), linestyle="-", linewidth=1,
-      color="black"
-  )
-  plt.scatter(
-      np.arange(1, 101), np.squeeze(imm_rates), color="red", marker="o", s=4
-  )
-  plt.grid(
-      visible=True, which='major', axis='both', color='0.5', linestyle='--',
-      linewidth=0.3
-  )
-  plt.xlabel(r"Age ($s$)")
-  plt.ylabel(r"Immigration rate ($i_s$)")
-  plt.savefig('imm_rates_zaf.png')
+  plt.savefig(os.path.join(plot_path, "imm_rates.png"), dpi=300)
   plt.show()
   ```
 
-  ```{figure} ./images/imm_rates_zaf.png
+  ```{figure} ./images/imm_rates.png
```

Comment:
@jdebacker. This image is missing. My solution is to just leave it that way for now because I couldn't get the `get_imm_rates()` function to plot them properly.